addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6588,15 +6588,17 @@ registry.ImageTools = ImageHandlerOption.extend({
             mimetype: this._getImageMimetype(img),
         });
 
-        await new Promise(resolve => {
-            this.$target.one('image_cropper_destroyed', async () => {
-                if (isGif(this._getImageMimetype(img))) {
-                    img.dataset[img.dataset.shape ? 'originalMimetype' : 'mimetype'] = 'image/png';
-                }
-                await this._reapplyCurrentShape();
-                resolve();
+        if (!imageCropWrapper.component._cropperClosed) {
+            await new Promise(resolve => {
+                this.$target.one('image_cropper_destroyed', async () => {
```

Comment:
Double quotes everywhere, as we are changing this code, can we change the code in vanilla JS?